background/ncn_plan_of_record.md

```diff
@@ -24,14 +24,25 @@ This document outlines the hardware necessary to meet CSM's Plan of Record (PoR)
 
 > **`NOTE:`** Several components below are necessary to provide redundancy in the event of hardware failure.
 
+Any of the disks may be used over the following busses:
```

Comment:
My understanding is that both "busses" and "buses" are technically correct, but that "buses" is considered the more standard spelling.

```suggestion
Any of the disks may be used over the following buses:
```